### DISCUSSION TITLE
Is there a simple way to get logits outputs from the CogVLM inference?

### DISCUSSION THREAD

**Discussion by aldoz-mila** [THUDM/CogVLM#381] [Q&A]
I am trying to get a better understanding of the confidence of the model and was trying to see if I can get the probabilistic output distribution.
